libs/core/src/main/java/org/opensearch/Version.java

```diff
@@ -146,7 +146,8 @@ public class Version implements Comparable<Version>, ToXContentFragment {
     public static final Version V_2_18_1 = new Version(2180199, org.apache.lucene.util.Version.LUCENE_9_12_1);
     public static final Version V_2_19_0 = new Version(2190099, org.apache.lucene.util.Version.LUCENE_9_12_1);
     public static final Version V_2_19_1 = new Version(2190199, org.apache.lucene.util.Version.LUCENE_9_12_1);
-    public static final Version CURRENT = V_2_19_1;
+    public static final Version V_2_19_2 = new Version(20190299, org.apache.lucene.util.Version.LUCENE_9_12_1);
```

Comment:
Is this supposed to have an extra 0 than the lines above? Looks like it should be `2190299`

Reply:
We are all fondly remembering 2019. :)

Reply:
@peterzhuamazon @gaiksaya FYI

Reply:
Oh I see what you mean this should be handled by core workflow?
https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/version.yml

Reply:
I'll push a commit to fix the number

Reply:
Pushed a small commit on each of the autocut PRs to correct the version here